### ISSUE TITLE
world crashing after updating to exnihiloomnia{1.2.2}

### ISSUE THREAD

**Issue by SDUBZ** [Jozufozu/ExNihiloOmnia#124]
[crash-2016-12-17_09.46.52-server.txt](https://github.com/Jozufozu/ExNihiloOmnia/files/658520/crash-2016-12-17_09.46.52-server.txt)

any help would be most useful

**Comment by Jozufozu** [comment]
That's strange it only happened after updating. It looks like a problem with RF tools

**Comment by Zarepheth** [comment]
Are you sure that version 1.2.2 only removes the Ex Nihilo Omnia specific gravels, sands, and dusts for ore? I'm thinking it does a whole lot more...

When I loaded up my world, I had molten metals flowing through my warehouse, my TConstruct smeltery's blocks were replaced with leaves, the Storage Drawers blocks were replaced with blocks from other mods, the Tinks Alloying Add-On's Vibranium ore blocks were replaced with Soul Glass... Things were a general mess! (Note: Forge warned me that the world's Backup save was being used - I assumed that was related to the Ex Nihilo Omnia updates...) I should have heeded the warning and backed up my world. It is very corrupted. Maybe the removal of those ore-related gravels, sands, and dusts messed up the table of item ids.

[crash-2016-12-16_18.18.40-client.txt](https://github.com/Jozufozu/ExNihiloOmnia/files/658538/crash-2016-12-16_18.18.40-client.txt)
